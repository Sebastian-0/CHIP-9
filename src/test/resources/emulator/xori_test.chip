ldi A, 0b01100110

# Zero flag
xori 0b01100110
sout A
sout FLAGS

# Negative flag
xori 0b10000001
sout A
sout FLAGS

# No flags
xori 0b10011001
sout A
sout FLAGS

hcf