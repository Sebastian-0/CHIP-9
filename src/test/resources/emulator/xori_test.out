0b00000000
128

0b10000001
64

0b00011000
0